masmini-common/src/main/java/masmini/Middleware.kt

```diff
@@ -0,0 +1,19 @@
+package masmini
+
+/**
+ * Middleware that will be called for every dispatch to modify the
```

Comment:
I'd not use `Middleware` to define a `Middleware`, `Class` or `Object` maybe?
```suggestion
 * Class that will be called for every dispatch to modify the
```

Reply:
Class +1